src/components/Input.tsx

```diff
@@ -0,0 +1,6 @@
+export default function Input({id, label,...props }) {
+    return <div className="flex flex-col">
+        {label && <label className="text-white mb-2 pointer-events-none" for={id}>{label}</label>}
```

Comment:
Curious on why we need `pointer-events-none`?

Reply:
Hovering on label, was triggering the hover of input field as well which i felt was not good

Reply:
yeah. But it hijacks the usecase of label. I think that's fine and it helps the users even more.

Reply:
Makes sense, will remove it